cdap-docs/developers-manual/source/building-blocks/workflows.rst

```diff
@@ -432,30 +477,31 @@ Taking our first example and modifying it, you could use code such as::
 where ``MyPredicate`` is a public class which implements the ``Predicate`` interface as::
 
   public static class MyPredicate implements Predicate<WorkflowContext> {
-
     @Override
     public boolean apply(@Nullable WorkflowContext input) {
```

Comment:
I think it would be better to rename the parameter "input" to "context" or "workflowContext"